src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4830,6 +4835,15 @@ private bool IsUsingAliasInScope(string name)
                 return BadExpression(node, boundLeft);
             }
 
+            // PROTOTYPE(default) unify with case above
+            // No member accesses on default
```

Comment:
I would expect this case (and the following one handling UnboundLambda) to be handled in the call to `MakeMemberAccessValue`. That call should never return a typeless expression.